sky/cli.py

```diff
@@ -114,6 +114,18 @@ def _get_glob_clusters(clusters: List[str]) -> List[str]:
     return list(set(glob_clusters))
 
 
+def _get_glob_storages(storages: List[str]) -> List[str]:
+    """Returns a list of storages that match the glob pattern."""
+    glob_storages = []
+    for storage_object in storages:
+        glob_storage = global_user_state.get_handle_from_storage_name(
```

Comment:
UX nit: If the number of storages to be deleted is >1, can we print the number of storages that matched? Something like "Deleting {count} storage objects."